src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -370,7 +370,6 @@ Task<TResult> IDurableOrchestrationContext.CallActivityWithRetryAsync<TResult>(s
             {
                 throw new ArgumentNullException(nameof(retryOptions));
             }
-
```

Comment:
nit: I think our style cop configuration will yell at this. This will create a warning, which will fail on release builds.

Reply:
I see, thanks for the suggestion. Is there a way I can locally run the style cop? So that I can test it in the future.

I'll add the new line :)

Reply:
In fact, I'm adding newlines after every `{}`, since the editor displays a warning / green swiggly line otherwise.

Reply:
Yep style-cop integrates very nicely with VS, (and probably VS Code, though I haven't tried that).